frontend/src/users/components/Navbar3.js

```diff
@@ -1,85 +1,115 @@
+                        <button className="navbar-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar" aria-controls="offcanvasNavbar" aria-label="Toggle navigation">
+                        <span className="navbar-toggler-icon ham"></span>
+                        </button>
+                        <div className="offcanvas offcanvas-end" tabIndex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
+                        <div className="offcanvas-header">
+                            <h5 className="offcanvas-title" id="offcanvasNavbarLabel">Night Canteen</h5>
+                            <button type="button" className="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"></button>
+                        </div>
+                        <div className="offcanvas-body">
+                            <ul className="navbar-nav justify-content-end flex-grow-1 pe-3">
+                            <li className="nav-item">
+                                <a className="nav-link active" aria-current="page" href="/">Home</a>
+                            </li>
+                            <li className="nav-item">
+                                <a className="nav-link" href="/">Menu</a>
+                            </li>
+                            <li className="nav-item">
+                                <a className="nav-link" href="/">About</a>
+                            </li>
+                            <li className="nav-item">
+                                <a className="nav-link" href="/">Services</a>
+                            </li>
+                            <li className="nav-item">
+                                <a className="nav-link" href="/">Log In</a>
+                            </li>
+                            <li className="nav-item">
+                                <a className="nav-link" href="/">Sign In</a>
+                            </li>
+                            
+                            </ul>
+                            
+                        </div>
+                        </div>
+                    </div> */}
+                    
+                
+        {/* </div> */}
         <nav className="navbar ofc fixed-top media">
             <div className="container-fluid">
                     <a className="navbar-brand" href="/"><b>Night Canteen</b></a>
                     <button className="navbar-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar" aria-controls="offcanvasNavbar" aria-label="Toggle navigation">
                     <span className="navbar-toggler-icon"></span>
                     </button>
-                    <div className="offcanvas offcanvas-end" tabIndex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
+                    <div className="offcanvas offcanvas-end" tabindex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
```

Comment:
Here it should have been `tabIndex` only.
`tabindex` in regular HTML becomes `tabIndex` in React HTML (JSX).

That is also the reason I was seeing many of the console errors in your screenshots : )